### DISCUSSION TITLE
Lost all account projects from Github account

### DISCUSSION THREAD

**Discussion by andre-rd-rodrigues** [vercel/community#6582] [Account Issue]
### Question

I've opened a new account with this email: barbara.barbizani.caiado@gmail.com. After connecting my github with the account, now all my projects from **andrerdrodrigues** github account are gone. I can't access them, when I login with github, it redirects me to this new account and I've no projects there. How can I revert this and access my projects from github?

<img width="1382" alt="Captura de ecrã 2024-04-21, às 19 37 42" src="https://github.com/vercel/community/assets/67346660/b1f6d338-458b-4e0b-aa68-d498e73eca95">

**Comment by amyegan** [comment]
Hey @andre-rd-rodrigues. Only one Vercel account can be connected to a GitHub account, and new connections will remove older ones. You can still access your other account if you login with email instead of using the GitHub login button. Just use the primary email address from your GitHub account.
